react/features/invite/components/add-people-dialog/native/AddPeopleDialog.js

```diff
@@ -169,15 +212,7 @@ class AddPeopleDialog extends AbstractAddPeopleDialog<Props, State> {
         }
 
         return (
-            <JitsiModal
-                footerComponent = { this._renderShareMeetingButton }
-                headerProps = {{
-                    forwardDisabled: this._isAddDisabled(),
-                    forwardLabelKey: 'inviteDialog.send',
-                    headerLabelKey: 'inviteDialog.header',
-                    onPressForward: this._onInvite
-                }}
-                modalId = { ADD_PEOPLE_DIALOG_VIEW_ID }>
+            <>
```

Comment:
Why is this not enclosed in a JitsiScreen?

Reply:
It doesn't need a wrapper.

Reply:
Same as above, I think we do need the safearea view and also the keyboard avoidance that the wrapper gives u. In general I think it's a good idea to have that baseline for all modal screens.